web/satellite/src/components/project/UsageReport.vue

```diff
@@ -161,13 +161,15 @@ import { toUnixTimestamp } from '@/utils/time';
                     });
                 },
                 onReportClick: function (): void {
-                    let projectID = this.$store.getters.selectedProject.id;
+                    const projectID = this.$store.getters.selectedProject.id;
+                   	const startDate = this.$store.state.usageModule.startDate;
+					const endDate = this.$store.state.usageModule.endDate;
```

Comment:
The Idents look off to me

Reply:
isn't there a linter for this part of the code which could check this kind of things?

Reply:
Yeah, that is something with my IDE it shows idents in a correct place even though they are not)
There is one but it isn't integrated with travis

Reply:
Let's open an issue in Jira for it with low priority